shelley/chain-and-ledger/formal-spec/protocol-parameters.tex

```diff
@@ -143,6 +143,7 @@ \section{Protocol Parameters}
       \StartRewards & \Duration & \text{duration to start reward calculations}\\
       \MaxKESEvo & \N & \text{maximum KES key evolutions}\\
       \Quorum & \N & \text{quorum for update system votes}\\
+      \MaxPV & \ProtVer & \text{all blocks are invalid after this value}\\
```

Comment:
I am not sure I understand this, I think there is a bit of context that should be added

Reply:
yea, I should add some context somewhere.

Reply:
ok, I added some context in the paragraphs about the globals in the protocol parameters section. While do so, I realized that in fact this cap is supposed to be for the major version only, so I made that change as well (ie not a max protocol version, but a max _major_ protocol version).